keycloak/keycloak-admin-ui
# Ream-settings -> User Profile -> Edit Attribute

## Motivation
Edit attribute functionality as per the old console.

Bugs found during dev, to be addressed in the future PRs related to edit functionality:
1. General Settings section: "Enabled when" doesn't show correct radio button selection and the "Required" section has the same issue with its radio buttons in editMode
2. Permissions section: Cheboxes don't show correct selection in editMode

## Verification Steps
1. Navigate to realm-settings -> User Profile tab (available when User Profile enabled in General tab)
2. Create a new attribute. 
3. Select created attribute (either by selecting the link in the table or the 'Edit' option more dropdown) and edit all or some of the fields.
4. Verify that the attribute was successfully edited, but keep in mind the bugs found above.

## Checklist:
